{"name": "Заказы пользователя из раздела «История заказов» отображаются на странице «Лента заказов»,", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <order_feed_test.TestsOrderFeed object at 0x000001FC9A8A7B90>, driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"59b0d5ad-2385-458b-b95e-a4a1f524f96c\")>\nuser = {'email': 'oedtzknlpm@yandex.ru', 'json': {'accessToken': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY1ZTc...True, 'user': {'email': 'oedtzknlpm@yandex.ru', 'name': 'oedtzknlpm'}}, 'name': 'oedtzknlpm', 'password': 'kdavugqvtm'}\n\n    @allure.title('Заказы пользователя из раздела «История заказов» отображаются на странице «Лента заказов»,')\n    def test_order_in_feed_and_order_history(self, driver, user):\n        order_feed_page = OrderFeedPage(driver)\n        order_feed_page.go_to_site(Url.LOGIN_URL)\n        personal_account_page = PersonalAccountPage(driver)\n        personal_account_page.login_to_account(user)\n        main_function_page = MainFunctionPage(driver)\n        main_function_page.add_ingredient()\n        main_function_page.click_place_order()\n>       num_order = order_feed_page.get_num_order()\n\ntests\\order_feed_test.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\order_feed_page.py:19: in get_num_order\n    self.waiting_for_the_order_number(OrderFeedLocators.NUM_ORDER, '9999')\npages\\base_page.py:42: in waiting_for_the_order_number\n    return WebDriverWait(self.driver, time).until_not(EC.text_to_be_present_in_element(locator, template))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"59b0d5ad-2385-458b-b95e-a4a1f524f96c\")>, method = <function text_to_be_present_in_element.<locals>._predicate at 0x000001FC9A9A3E20>, message = ''\n\n    def until_not(self, method: Callable[[D], T], message: str = \"\") -> Union[T, Literal[True]]:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value evaluates to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`, or\n                  ``True`` if `method` has raised one of the ignored exceptions\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if not value:\n                    return value\n            except self._ignored_exceptions:\n                return True\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:128: TimeoutException"}, "steps": [{"name": "Вход в аккаунт", "status": "passed", "parameters": [{"name": "data", "value": "{'email': 'oedtzknlpm@yandex.ru', 'name': 'oedtzknlpm', 'password': 'kdavugqvtm', 'json': {'success': True, 'user': {'email': 'oedtzknlpm@yandex.ru', 'name': 'oedtzknlpm'}, 'accessToken': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY1ZTcwYzljOWVkMjgwMDAxYjNjMzFhYSIsImlhdCI6MTcwOTY0MDg2MCwiZXhwIjoxNzA5NjQyMDYwfQ.pG35f5bim6hQC-P60r4Y1HzkmBpPUET3j3R2jQ6a0ok', 'refreshToken': '3dab0729cf9d0db7eee05406baf551ec310860beaf0eaa3d35b4bf0733b3bd8436486d387685cf95'}}"}], "start": 1709640862633, "stop": 1709640863222}, {"name": "Добавление ингредиента", "status": "passed", "start": 1709640863222, "stop": 1709640864320}, {"name": "Клик 'Оформить заказ'", "status": "passed", "start": 1709640864320, "stop": 1709640864585}, {"name": "Получить номер заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"D:\\Python\\Diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"D:\\Python\\Diplom_3\\pages\\order_feed_page.py\", line 19, in get_num_order\n    self.waiting_for_the_order_number(OrderFeedLocators.NUM_ORDER, '9999')\n  File \"D:\\Python\\Diplom_3\\pages\\base_page.py\", line 42, in waiting_for_the_order_number\n    return WebDriverWait(self.driver, time).until_not(EC.text_to_be_present_in_element(locator, template))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Python\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 128, in until_not\n    raise TimeoutException(message)\n"}, "start": 1709640864586, "stop": 1709640875076}], "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1709640860527, "stop": 1709640875080, "uuid": "b9ab8ae4-132b-48f8-a725-37571ff6d927", "historyId": "70cbf6330c3285feefa5c20cf7be5b9d", "testCaseId": "cb1583a297e0cfe2194a5b50721795a4", "fullName": "tests.order_feed_test.TestsOrderFeed#test_order_in_feed_and_order_history", "labels": [{"name": "story", "value": "Тест 'Лента Заказов'"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "order_feed_test"}, {"name": "subSuite", "value": "TestsOrderFeed"}, {"name": "host", "value": "oz1"}, {"name": "thread", "value": "80332-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.order_feed_test"}]}